dut/drivers.wake

```diff
@@ -0,0 +1,59 @@
+
+tuple SkeletalDriverPlan =
+  global DuhFile:             Path
+  global CompanyName:         String
+  global DeviceName:          String
+  global LocalMetalDirectory: String
+  global DUTPlan:             RocketChipDUTPlan
+
+global def makeSkeletalDriverPlan = SkeletalDriverPlan _ _ _ _ _
+
+def getDevicePlan deviceName =
+  def plans = subscribe skeletalDriverPlans
+  def samePlan aPlan =
+    if aPlan.getSkeletalDriverPlanDeviceName ==~ deviceName
+    then Some aPlan
+    else None
+  mapPartial samePlan plans
+  | head
+
+def skeletalPlanToPlan skeletalDriverPlan =
+  def objModel =
+    makeRTL (skeletalDriverPlan.getSkeletalDriverPlanDUTPlan)
+    | getDUTObjectModel
+    | prettyJSON
+
+  def name =
+    skeletalDriverPlan
+    | getSkeletalDriverPlanDUTPlan
+    | getRocketChipDUTPlanName
+
+  def omfile = write "{workspace}/build/api-generator-sifive/{name}/objectmodel.json" objModel
+  def cmd =
+    (which "python3.7"),
```

Comment:
We shouldn't be using `which`. My understanding is that this will bypass the runner system by evaluating the path to the `python3.7` executable prior to handing it off to runner.

Reply:
removed